Capri.Services/Courses/CourseUpdater.cs

```diff
@@ -32,6 +36,12 @@ public class CourseUpdater : ICourseUpdater
                     $"Course with id {id} does not exist");
             }
 
+            var facultyResult = await _facultyGetter.Get(newData.FacultyId);
+            if(!facultyResult.Successful())
```

Comment:
Where is it used? because I don't see any reference later in that method.

Reply:
This is for checking if such a faculty exists. If not, it returns those errors. Otherwise we would get a database error